src/components/experiment/ExperimentPage.tsx

```diff
@@ -0,0 +1,24 @@
+import { Grid } from '@mui/material';
+import React from 'react';
+
+import { StyledContainer, StyledPaper } from 'styles/StyledComponents';
+
+import ExperimentFilterBar from './ExperimentFilterBar';
+import ExperimentTable from './ExperimentTable';
+
+function ExperimentPage() {
+  return (
+    <StyledContainer>
+      <Grid container>
```

Comment:
Grid container and item not needed here as we removed them to follow some better standard for all pages.